_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,424 @@
+// ------------------------------
+// edX Pattern Library: Components - Forms
+
+// About: Contains base styling for forms
+
+// #GLOBAL
+// #INPUT TEXT
+// #INPUT RADIO/CHECKBOX
+// #SELECT
+// #TEXTAREA
+// #SLIDERS
+// #PROGRESS
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.fieldset {
+    @include clearfix;
+    margin-bottom: spacing-vertical(mid-small);
+
+    .group-legend {
+        margin-bottom: spacing-vertical(x-small);
+    }
+
+    .field-wrapper {
+        margin-bottom: spacing-vertical(base);
+
+        .helper-text {
+            font-size: font-size(small);
+            color: $helper-text;
+        }
+    }
+
+    &.form-group {
+
+        .field-wrapper { // Generally used for radios and checkboxes
+            margin-bottom: spacing-vertical(x-small);
+        }
+
+        .helper-text { // Generally used for radios and checkboxes
+            margin-top: 0;
+            margin-left: 24px;
+        }
+    }
+}
+
+.form-label {
+    display: block;
+    margin-bottom: spacing-vertical(x-small);
+    color: $label-color;
+
+    &.inline-label {
+        margin-bottom: 0;
+    }
+
+    &.is-active,
+    &.is-selected {
+        color: $label-color-active;
+    }
+}
+
+.helper-text {
+    margin-top: spacing-vertical(x-small);
+}
+
+.required-field:after {
+    display: inline-block;
+    margin-left: ($baseline/4) + px;
+    content: "*";
+    font-size: font-size(small);
+    color: $required-indicator-color;
```

Comment:
In the designs, this is the same color as the label text. Not sure it needs a special variable either.